libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -62,6 +62,7 @@
 #define HAL_BOARD_SUBTYPE_CHIBIOS_VRUBRAIN_V51  5018
 #define HAL_BOARD_SUBTYPE_CHIBIOS_VRCORE_V10    5019
 #define HAL_BOARD_SUBTYPE_CHIBIOS_VRBRAIN_V54   5020
+#define HAL_BOARD_SUBTYPE_CHIBIOS_SUCCEXF4      5021
```

Comment:
this is not needed any more, it is only needed if you need an #ifdef somewhere in the code that is specific to this board, which is rare these days